systems/controllers/osc/operational_space_control.cc

```diff
@@ -226,16 +241,24 @@ void OperationalSpaceControl::Build() {
   }
 
   // Construct QP
-  n_h_ = distance_constraints_.size();
-  n_c_ = 3 * body_indices_.size();
   prog_ = std::make_unique<MathematicalProgram>();
 
+  // Size of decision variable
+  n_h_ = (kinematic_evaluators_ == nullptr)
+             ? 0
```

Comment:
formatting looks weird here, I would think `? 0 : kinematic_evaluators_->count_full()` would be on the same line

Reply:
I'm not sure what the correct format is, but I changed it to
```
  n_h_ = (kinematic_evaluators_ == nullptr)?
      0 : kinematic_evaluators_->count_full();
```